pandas/core/indexes/base.py

```diff
@@ -681,7 +681,53 @@ def _values(self):
         return self.values
 
     def get_values(self):
-        """ return the underlying data as an ndarray """
+        """
+        Return `Index` data as an `numpy.ndarray`.
+
+        In its functionality it is the same as `Series.get_values`,
```

Comment:
the extended description is not very useful here. you can just say that this is a wrapper around .values